backend/blueprints/spa_api/service_layers/replay/replay_positions.py

```diff
@@ -0,0 +1,78 @@
+import gzip
+import os
+
+import pandas as pd
+
+from typing import List, cast
+from flask import current_app
+
+from carball.analysis.utils import proto_manager, pandas_manager
+from backend.database.objects import Game, PlayerGame
+from .replay_player import ReplayPlayer
+from ..utils import sort_player_games_by_team_then_id
+from ...errors.errors import ReplayNotFound, ErrorOpeningGame
+
+
+class ReplayPositions:
+    def __init__(self, id_: str,
+                 ball, players, colors, names,
+                 frames):
+        self.id = id_
+
+        self.ball = ball
+        self.players = players
+        self.colors = colors
+        self.names = names
+        self.frames = frames
+
+    @staticmethod
+    def create_from_id(id_: str) -> 'ReplayPositions':
+        pickle_path = os.path.join(current_app.config['PARSED_DIR'], id_ + '.replay.pts')
+        gzip_path = os.path.join(current_app.config['PARSED_DIR'], id_ + '.replay.gzip')
+        replay_path = os.path.join(current_app.config['REPLAY_DIR'], id_ + '.replay')
+        if os.path.isfile(replay_path) and not os.path.isfile(pickle_path):
+            raise ReplayNotFound()
+
+        try:
+            with gzip.open(gzip_path, 'rb') as f:
+                data_frame = pandas_manager.PandasManager.safe_read_pandas_to_memory(f)
+            with open(pickle_path, 'rb') as f:
+                g = proto_manager.ProtobufManager.read_proto_out_from_file(f)
+        except Exception as e:
+            raise ErrorOpeningGame(str(e))
+
+        field_ratio = 5140.0 / 4120
+        x_mult = 100.0 / 4120
```

Comment:
Explain the need for this? (in a comment)

Reply:
This was pulled from the old replays endpoint - from what I understand, it was just to map the positions from field coordinates to [0, 100]

Reply:
Why not send the raw values now? This seems unnecessary and an additional part that can go wrong.

Reply:
Ah yeah, the code that actually uses these values is commented out - I'm removing it entirely in #119.